testing/src/scenario/_runtime.py

```diff
@@ -249,7 +136,7 @@ def _get_event_env(self, state: "State", event: "_Event", charm_root: Path):
                 else:
                     logger.warning(
                         "remote unit ID unset; no remote unit data present. "
-                        "Is this a realistic scenario?",
+                        "Is this a realistic scenario?",  # TODO: is it?
```

Comment:
Strong deja vu here haha, didn't we talk about this exact comment on another PR?

Reply:
Ah, dammit, maybe my rebase/merge has gone wonky and is showing stuff already in main. I checked it had the changes from the latest PR that was merged and that tests passed, but not more. I'll look.